server/utils/sftp.js

```diff
@@ -0,0 +1,121 @@
+import { printFuncError, checkDirectoryStatus } from '../utils/utils';
+const path = require('path');
+const client = require('ssh2-sftp-client');
+const { SFTP_IN_ENDPOINT, SFTP_HOST, SFTP_PORT, SFTP_USERNAME, SFTP_PASSWORD } = process.env;
+
+const connectionConfig = {
+    host: SFTP_HOST,
+    port: SFTP_PORT,
+    username: SFTP_USERNAME,
+    password: SFTP_PASSWORD,
+};
+
+// Unlike AWS and local file copy, SFTP requires the target file name
+// This additional function will include the generated filename in the target path
+export function getSFTPFilePath(src, dest) {
+    const filename = `/${path.basename(src)}`;
+    return dest + filename;
+}
+
+//Copy files to/from SFTP location
+export function fileCopy(src, dest) {
+    const sftp = new client();
+    return new Promise((resolve, reject) => {
+        sftp.connect(connectionConfig)
+            .then(() => {
```

Comment:
maybe to follow the suite?

```suggestion
            .then(() => sftp.put(src, getSFTPFilePath(src, dest)))
```

Reply:
updated.